mock-user-management.js

```diff
@@ -127,8 +127,9 @@ function getCredentialId(vp) {
 }
 
 function getCredentialType(vc) {
-  const types = vc.type;
-  return types.slice(1).join('/');
+  if (!vc) return 'Credential';
```

Comment:
Minor style quibble. I would also normally question not just throwing when a `vc` isn't given here, but it's simple demo code so ... I will avert my eyes. :)

```suggestion
  if(!vc) {
    return 'Credential'
  };
```